jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -119,6 +119,7 @@ public class OdeProperties implements EnvironmentAware {
    private Set<String> kafkaTopicsDisabledSet = new HashSet<>();
 
    // BSM
+   private final String BSM_START_FLAG = "0014";
```

Comment:
These strings were moved into the UperUtil class in #78 . Are they currently used anywhere from the OdeProperties class? If not I don't believe we should add these back in.

Reply:
Thanks for this! These have been removed & the unit tests have been verified to be passing still.